applications/serial_lte_modem/Kconfig

```diff
@@ -27,6 +27,10 @@ config SLM_AT_MAX_PARAM
 config SLM_NATIVE_TLS
 	bool "Use Zephyr mbedTLS"
 
+config SLM_CONFIRM_XSLEEP
+	bool "Send OK confirmation after the AT#XSLEEP=... command"
+	default n
```

Comment:
No need for `default n`, it's implicit behaviour.

Reply:
Good point, removed & thanks.